src/openforms/logging/tests/base.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from openforms.logging.models import TimelineLogProxy
+
+
+class LoggingTestMixin(unittest.TestCase):
+    def assertLogExtraDataEquals(self, log: TimelineLogProxy, **extra_data):
+        self.assertIsNotNone(log.extra_data)
+        self.assertNotEqual(extra_data, dict())
+
+        for key, value in extra_data.items():
+            self.assertIn(key, log.extra_data)
+            self.assertEqual(log.extra_data[key], value)
```

Comment:
```suggestion
        for key, value in extra_data.items():
            with self.subTest(key=key, value=value):
                self.assertIn(key, log.extra_data)
                self.assertEqual(log.extra_data[key], value)
```

Reply:
Reverted this since no more AssertionErrors are raised and the test testing the mixin then fails